From c303c3fc58e3606fff638c26bdff8ca0c1b80152 Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 22 Apr 2020 19:27:34 +0000
Subject: [PATCH 371/462] Mantis 56312: Extend Desko-Penta Device-Handler by
 Accufill-SDK for reading US driving-licenses

Build with new version of AssureID SDK

git-svn-id: svn://localhost/SelfServiceBuild/trunk@679 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index f5625c1..6b9dc0c 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -394,7 +394,7 @@ set DIR_SPEECH11=%DIR_GLOBALSLIB%\Microsoft_SDKs\Speech\v11.0
 set INC_SPEECH11=-I%DIR_SPEECH11%\Include
 set LIB_SPEECH11=%DIR_SPEECH11%\Lib\sapi.lib
 
-set DIR_ASSUREID=%DIR_GLOBALSLIB%\AssureID\sdk
+set DIR_ASSUREID=%DIR_GLOBALSLIB%\AssureID\5.2.61.40\sdk
 set INC_ASSUREID=-I%DIR_ASSUREID%\com -I%DIR_ASSUREID%\dotnet\4.0
 set LIB_ASSUREID=%DIR_ASSUREID%\dotnet\4.0\x86\AssureTec.AssureID.Platform.SDK.Extensions.lib
 
-- 
2.41.0.windows.1

